Electrical_Panels_EUSS/postprocess_electrical_panel_size_nec.py

```diff
@@ -133,18 +232,18 @@ def _general_load_kitchen(row, n=2):
     if n == "auto":
         n = 2  # start with min requirement
         # TODO: can expand based on building_type, vintage, and floor_area
+        """
         if (row["build_existing_model.misc_extra_refrigerator"] != "None") or (
             row["build_existing_model.misc_freezer"] != "None"
         ):
-            n += 2  # 2 additional branches added for misc refrigeration (outside kitchen)
-
+            n += 2  # 2 additional branches added for misc refrigeration (outside kitchen) #TODO: This is incorrect, small appliance branch circuits can only supply loads in the kitchen
```

Comment:
Can we back out kitchen sqft?

Reply:
receptacle is non-continuous, larger counter space increases # of receptacles (min 4-ft apart) and each branch circuit has a max # of receptacles it can support.